src/test/scala/checkOldVersions.scala

```diff
@@ -1,23 +1,41 @@
 package ohnosequences.db.ncbitaxonomy.test
 
-import ohnosequences.db.ncbitaxonomy.{Version, names, nodes}
+import ohnosequences.db.ncbitaxonomy._
+import org.scalatest.DoNotDiscover
 
+@DoNotDiscover
 class CheckOldVersions extends NCBITaxonomyTest("CheckOldVersions") {
 
-  test("S3 objects (nodes and names) exist for all versions") {
+  test("nodes.dmp and names.dmp exist in S3 for all versions") {
 
     val versionObjects =
       Version.all map { version =>
         (version, names(version), nodes(version))
       }
 
-    assert {
-      versionObjects forall {
-        case (version, namesObj, nodesObj) =>
-          println(s"S3 objects exist: ${version.name}")
-          objectExists(namesObj) &&
-          objectExists(nodesObj)
+    versionObjects foreach {
+      case (version, namesObj, nodesObj) =>
+        assert(
+          objectExists(namesObj) && objectExists(nodesObj),
+          s"S3 objects names.dmp and nodes.dmp do not exist for version ${version.name}"
+        )
+    }
+  }
+
+  test(
+    "data.tree and shape.tree exist in S3 for all versions. If not, create them") {
```

Comment:
This does not create anything, isn't it?

Reply:
Correct! I finally left all that work to the `mirror` test, which is ordered to be executed before this check